src/Cli/dotnet/NugetPackageDownloader/NuGetPackageDownloader.cs

```diff
@@ -131,15 +134,18 @@ internal class NuGetPackageDownloader : INuGetPackageDownloader
 
         private void VerifySigning(string nupkgPath)
         {
-            if (!_verifySignatures)
+
+            if (!_verifySignatures && !_validationMessagesDisplayed)
             {
-                if (!_validationMessagesDisplayed)
+                if (_verbosityOptions != VerbosityOptions.quiet && _verbosityOptions != VerbosityOptions.q)
```

Comment:
It will be great to have no `LocalizableStrings.NuGetPackageSignatureVerificationSkipped` message in `minimal` verbosity apart from `quiet` verbosity.

Reply:
Done